plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -89,6 +89,9 @@ export default class JobsOverviewTable extends React.Component {
   getData(data) {
     return Object.values(
       data.nodes.reduce((acc, job) => {
+        if (job.path.length < data.path.length) {
```

Comment:
Some people will be against such a bug fix and will be in favour of refactoring. We should refactor this method, sure. But for me, it's more important to fix the bug right now (especially since it was reported more than once recently), and separate the bug fixing and the refactoring as we agreed to do. We can always focus on just refactoring.

Reply:
I'm good with refactoring later. Could you create a ticket for that?

Reply:
Yes, I will.

Reply:
https://jira.mesosphere.com/browse/DCOS-57424